upcase_booklist_app/templates/user.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer | Users </title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body>
+        <nav class="page-header">
+            <div class="header-items"><a href="{{ url_for('index') }}">Home</a></div> 
+            <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
+            <div class="header-items"><a href="{{ url_for('authors') }}">Authors</a></div> 
+            <div class="header-items"><a href="{{ url_for('lists') }}">Lists</a></div>
+            <div class="header-items"><a href="{{ url_for('users') }}">Users</a></div>
+        </nav>
+
+        <div>
+            <h1>
+                Curious Programmer's Users
+            </h1>
+        </div>
+
+        <div>
+            <div class="container-user-name">
```

Comment:
The entire structure of this page looks really good, starting with the `h1`, and followed a list of users.